cmd/vic-machine/configure/configure.go

```diff
@@ -382,6 +391,22 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 		return err
 	}
 
+	// Handle the three options for the --affinity-vm-group flag: unset, true, false.
+	//
+	// If the user hasn't specified the flag, we don't want to make a change. If they have
+	// specified it and are requesting a change, track that.
+	if clic.IsSet("affinity-vm-group") {
```

Comment:
it seems like this parameter has annotation, why do you access it by name?

Reply:
There are three possible states:

1. The user has not specified `--affinity-vm-group`.
2. The user has specified `--affinity-vm-group=false`.
3. The user has specified `--affinity-vm-group` or `--affinity-vm-group=true`.

As a boolean, `UseVMGroup` can only represent two states, so it treats 1 and 2 as equivalent.

For our purposes, the distinction is significant: 1 means do nothing and 2 means to disable. To distinguish between this cases, we first check `IsSet`; if this returns `false`, we're in case 1.